### PR TITLE
fix: windows native method need default result

### PR DESCRIPTION
**Pull request by wangbax**
## :scroll: Description

When flutter is natively connected, the unimplemented logic needs to call back `NotImplemented` by default, otherwise it will be stuck in `await`.

## :bulb: Motivation and Context

Pending line: https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/sentry_tracer.dart#L102
Link code: [sentry_native_channel](https://github.com/getsentry/sentry-dart/blob/main/flutter/lib/src/sentry_native_channel.dart)

## :green_heart: How did you test it?
- Init set `SentryFlutter.init.options.debug = true`
- Check log print

image

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps
no op

_#skip-changelog_

### CODE REVIEW

File: flutter/windows/sentry_flutter_plugin.cpp [hunk 1]
```diff
@@ -57,6 +57,7 @@ void SentryFlutterPlugin::HandleMethodCall(
     const flutter::MethodCall<flutter::EncodableValue> &method_call,
     std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
   // Native features will be added in a next release
+  result->NotImplemented();
```
Comment (marandaneto) [inline flutter/windows/sentry_flutter_plugin.cpp:60]: Do we need that for Linux too?

See `sentry_flutter_plugin_handle_method_call`

Comment (wangbax) [reply flutter/windows/sentry_flutter_plugin.cpp:60]: Actually, already processed on linux. ☺️

[fl_method_not_implemented_response_new](https://github.com/getsentry/sentry-dart/blob/main/flutter/linux/sentry_flutter_plugin.cc#L25)

**Comment by marandaneto** [comment]
@wangbax please add a changelog entry :) it's a user facing fix.

**Comment by wangbax** [comment]
@marandaneto has done. 😊

**Review by marandaneto — APPROVED** [review]

**Pull request merged**